python/py-eli5/Portfile

```diff
@@ -0,0 +1,99 @@
+                    undoing hashing\; Keras - explain predictions of\
+                    image classifiers via Grad-CAM visualizations\;\
+                    xgboost - show feature importances and explain\
+                    predictions of XGBClassifier, XGBRegressor and\
+                    xgboost.Booster\; LightGBM - show feature\
+                    importances and explain predictions of\
+                    LGBMClassifier and LGBMRegressor\; CatBoost - show\
+                    feature importances of CatBoostClassifier,\
+                    CatBoostRegressor and catboost.CatBoost\; lightning\
+                    - explain weights and predictions of lightning\
+                    classifiers and regressors\; sklearn-crfsuite. ELI5\
+                    allows to check weights of sklearn_crfsuite.CRF\
+                    models. ELI5 also implements several algorithms\
+                    for inspecting black-box models (see Inspecting\
+                    Black-Box Estimators): TextExplainer allows to\
+                    explain predictions of any text classifier using\
+                    LIME algorithm (Ribeiro et al., 2016). There are\
+                    utilities for using LIME with non-text data and\
+                    arbitrary black-box classifiers as well, but this\
+                    feature is currently experimental\; Permutation\
+                    importance method can be used to compute feature\
+                    importances for black box estimators. Explanation\
+                    and formatting are separated\; you can get\
+                    text-based explanation to display in console, HTML\
+                    version embeddable in an IPython notebook or web\
+                    dashboards, a pandas.DataFrame object if you want\
+                    to process results further, or JSON version which\
+                    allows to implement custom rendering and\
+                    formatting on a client.
+
+checksums           rmd160  cbdd276878809cd586e3f93e759dd8eab5cfe155 \
+                    sha256  8e5e8ea78978137715e707c07df335647601bea65365e8b12b907cf0f5f69c21 \
+                    size    6821953
+
+python.versions     35 36 37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-attrs \
+                    port:py${python.version}-graphviz \
+                    port:py${python.version}-jinja2 \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-scikit-learn \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-singledispatch \
+                    port:py${python.version}-tabulate
+
+    depends_test-append \
+                    port:py${python.version}-tensorflow \
+                    port:py${python.version}-tox
```

Comment:
you seem to be missing dependencies here comparing to upstream's list

Reply:
I just deleted the whole test block.